### PR TITLE
Disable "new workspace" button when user is not logged in

### PR DESCRIPTION
**Pull request by waxlamp**
This PR also removes the "intro" popover, as (1) it has become sort of annoying and (2) its advice no longer applies.

Since the way to really get started with this app is to create an account and log in, I'd like to find a UI design principle that makes the login mechanism more obvious and inviting (currently it's hidden behind a general "user account" badge).

### CODE REVIEW

**Comment by waxlamp** [comment]
> There appears to be a new bug in this PR. When I click create workspace after refreshing (and on logging in, since it redirects to the home page) the modal appears for a split second and disappears. It's only persistent and usable on a second click. Let me know if the same behavior is happening for you.
> 
> I verified that this happens on chromium and firefox for me on my linux box, and on chrome on my mac.

Can you paste in any console errors that appear when this happens?

**Comment by JackWilb** [comment]
> Can you paste in any console errors that appear when this happens?

There are no errors in the console, but I think I've tracked down the issue to the v-menu having the `v-model="popover"` set on it. Removing it seems to fix the error.

**Comment by waxlamp** [comment]
> > Can you paste in any console errors that appear when this happens?
> 
> There are no errors in the console, but I think I've tracked down the issue to the v-menu having the `v-model="popover"` set on it. Removing it seems to fix the error.

Thanks for tracking this down. I pushed cdc4d95112821dd7693e6569a45279a20272fc65 to fix it; can you verify that it's working now?

File: src/components/WorkspaceDialog.vue [hunk 1]
```diff
@@ -15,12 +14,17 @@
         width="500"
       >
         <template v-slot:activator="{ on }">
+          <!-- See
+            https://github.com/vuetifyjs/vuetify/issues/4482#issuecomment-476689473
+            for why the `dark` prop is tied to the disabled state of the button.
+          -->
           <v-btn
             id="add-workspace"
+            :disabled="userInfo == null"
```
Comment (AlmightyYakob) [inline src/components/WorkspaceDialog.vue:23]:
```suggestion
            :disabled="userInfo === null"
```

Comment (waxlamp) [reply src/components/WorkspaceDialog.vue]: Thanks. I'm surprised the linter didn't catch this...

**Pull request merged**